### PR TITLE
[更新完毕] global setting

### PR DESCRIPTION
**Pull request by ArkhamWJZ**

### CODE REVIEW

File: locales/zh_CN/LC_MESSAGES/global_settings.po [hunk 1]
```diff
@@ -27,184 +28,179 @@ msgid "Backends"
 msgstr "后端"
 
 #: ../../global_settings.rst:7
-#, fuzzy
 msgid "To specify which Arch to use: ``ti.init(arch=ti.cuda)``."
-msgstr "指定要使用的Arch，请执行以下操作： ``export TI_ARCH=cuda`` 或 ``ti.init(arch=ti.cuda)``。"
+msgstr "指定要使用的 Arch，请使用： ``ti.init(arch=ti.cuda)``。"
```
Comment (archibate) [inline locales/zh_CN/LC_MESSAGES/global_settings.po]:
```suggestion
msgstr "指定要使用的后端，请使用： ``ti.init(arch=ti.cuda)``。"
```

File: locales/zh_CN/LC_MESSAGES/global_settings.po [hunk 2]
```diff
@@ -27,184 +28,179 @@ msgid "Backends"
...
+"init(device_memory_GB=0.5)``."
+msgstr "指定 CUDA 预分配内存的大小： ``ti.init(device_memory_GB=0.5)``。"
 
 #: ../../global_settings.rst:9
 msgid ""
-"To disable unified memory usage on CUDA: "
-"``ti.init(use_unified_memory=False)``."
-msgstr ""
+"To disable unified memory usage on CUDA: ``ti.init(use_unified_memory=False)``."
+msgstr "禁用 CUDA 使用统一内存：``ti.init(use_unified_memory=False)``。"
```
Comment (archibate) [inline locales/zh_CN/LC_MESSAGES/global_settings.po]:
```suggestion
msgstr "禁止 CUDA 使用统一内存 (Unified Memory)：``ti.init(use_unified_memory=False)``。"
```

File: locales/zh_CN/LC_MESSAGES/global_settings.po [hunk 3]
```diff
@@ -27,184 +28,179 @@ msgid "Backends"
...
-"To specify which GPU to use for CUDA: ``export "
-"CUDA_VISIBLE_DEVICES=[gpuid]``."
-msgstr "指定 CUDA 所使用的 GPU，请执行以下操作：``export CUDA_VISIBLE_DEVICES=[gpuid]``。"
+"To specify which GPU to use for CUDA: ``export CUDA_VISIBLE_DEVICES=[gpuid]``."
+msgstr "指定 CUDA 所使用的 GPU：``export CUDA_VISIBLE_DEVICES=[gpuid]``。"
 
 #: ../../global_settings.rst:11
 msgid "To disable a backend on start up, say, CUDA: ``export TI_ENABLE_CUDA=0``."
-msgstr ""
+msgstr "要禁用 CUDA 后端的启动：``export TI_ENABLE_CUDA=0``。"
```
Comment (archibate) [inline locales/zh_CN/LC_MESSAGES/global_settings.po]:
```suggestion
msgstr "要在启动时禁用某个后端，比如 CUDA：``export TI_ENABLE_CUDA=0``。"
```

File: locales/zh_CN/LC_MESSAGES/global_settings.po [hunk 4]
```diff
@@ -27,184 +28,179 @@ msgid "Backends"
...
 #: ../../global_settings.rst:19
 msgid "To show pretty Taichi-scope stack traceback: ``ti.init(excepthook=True)``."
-msgstr ""
+msgstr "如要显示优雅的 Taichi 作用域栈回溯: ``ti.init(excepthook=True)``。"
 
 #: ../../global_settings.rst:20
-#, fuzzy
 msgid "To print intermediate IR generated: ``ti.init(print_ir=True)``."
-msgstr "要打印生成的中间表示码： ``export TI_PRINT_IR=1`` 或 ``ti.init(print_ir=True)``。"
+msgstr "如要打印生成的中间表示码：``ti.init(print_ir=True)``。"
```
Comment (archibate) [inline locales/zh_CN/LC_MESSAGES/global_settings.po]:
```suggestion
msgstr "如要打印生成的中间表示码 (IR)：``ti.init(print_ir=True)``。"
```

File: locales/zh_CN/LC_MESSAGES/global_settings.po [hunk 5]
```diff
@@ -27,184 +28,179 @@ msgid "Backends"
...
 msgid "Runtime"
 msgstr "运行"
 
 #: ../../global_settings.rst:25
 msgid ""
-"Restart the entire Taichi system (destroy all tensors and kernels): "
-"``ti.reset()``."
+"Restart the entire Taichi system (destroy all tensors and kernels): ``ti."
+"reset()``."
 msgstr "重新启动 Taichi 运行系统（销毁所有张量和内核）： ``ti.reset()``。"
```
Comment (archibate) [inline locales/zh_CN/LC_MESSAGES/global_settings.po]:
```suggestion
msgstr "重新启动 Taichi 运行时系统（销毁所有张量和内核）： ``ti.reset()``。"
```

File: locales/zh_CN/LC_MESSAGES/global_settings.po [hunk 6]
```diff
@@ -27,184 +28,179 @@ msgid "Backends"
...
 "To start program in debug mode: ``ti.init(debug=True)`` or ``ti debug "
 "your_script.py``."
-msgstr "以调试模式启动程序，请执行以下操作： ``export TI_DEBUG=1`` 或 ``ti.init(debug=True)``。"
+msgstr ""
+"如要以调试模式启动程序：``ti.init(debug=True)`` 或 ``ti debug your_script.py``."
 
 #: ../../global_settings.rst:27
 msgid "To disable importing torch on start up: ``export TI_ENABLE_TORCH=0``."
-msgstr ""
+msgstr "禁用在启动时导入 torch: ``export TI_ENABLE_TORCH=0``。"
```
Comment (archibate) [inline locales/zh_CN/LC_MESSAGES/global_settings.po]:
```suggestion
msgstr "禁止在启动时导入 torch: ``export TI_ENABLE_TORCH=0``。"
```

File: locales/zh_CN/LC_MESSAGES/global_settings.po [hunk 7]
```diff
@@ -27,184 +28,179 @@ msgid "Backends"
...
 #: ../../global_settings.rst:32
-#, fuzzy
 msgid ""
-"Show more detailed log to level TRACE: ``ti.init(log_level=ti.TRACE)`` or"
-" ``ti.set_logging_level(ti.TRACE)``."
+"Show more detailed log to level TRACE: ``ti.init(log_level=ti.TRACE)`` or ``ti."
+"set_logging_level(ti.TRACE)``."
 msgstr ""
-"通过输出 TRACE 级日志以显示更详尽的信息：``export TI_LOG_LEVEL=trace`` 或 "
+"通过输出 TRACE 级日志以显示更详尽的信息：``ti.init(log_level=ti.TRACE)`` or "
```
Comment (archibate) [inline locales/zh_CN/LC_MESSAGES/global_settings.po]:
```suggestion
"通过输出 TRACE 级日志以显示更详尽的信息：``ti.init(log_level=ti.TRACE)`` 或 "
```

File: locales/zh_CN/LC_MESSAGES/global_settings.po [hunk 8]
```diff
@@ -27,184 +28,179 @@ msgid "Backends"
...
 
 #: ../../global_settings.rst:36
 msgid "Develop"
 msgstr "开发"
 
 #: ../../global_settings.rst:38
-#, fuzzy
 msgid "To trigger GDB when Taichi crashes: ``ti.init(gdb_trigger=True)``."
-msgstr "当 Taichi 崩溃时触发 GDB：``export TI_GDB_TRIGGER=1``。"
+msgstr "当 Taichi 崩溃时触发 GDB：``ti.init(gdb_trigger=True)``。"
```
Comment (archibate) [inline locales/zh_CN/LC_MESSAGES/global_settings.po]:
```suggestion
msgstr "如要在 Taichi 崩溃时触发 GDB：``ti.init(gdb_trigger=True)``。"
```

File: locales/zh_CN/LC_MESSAGES/global_settings.po [hunk 9]
```diff
@@ -27,184 +28,179 @@ msgid "Backends"
...
 #: ../../global_settings.rst:39
 msgid ""
-"Cache compiled runtime bitcode in **dev mode** to save start up time: "
-"``export TI_CACHE_RUNTIME_BITCODE=1``."
+"Cache compiled runtime bitcode in **dev mode** to save start up time: ``export "
+"TI_CACHE_RUNTIME_BITCODE=1``."
 msgstr ""
-"以 **开发模式** 缓存编译后运行时位码(compiled runtime bitcode)以节省启动时间：``export "
-"TI_CACHE_RUNTIME_BITCODE=1``。"
+"以 **开发模式** 缓存编译后运行时位码(compiled runtime bitcode)以节省启动时间："
```
Comment (archibate) [inline locales/zh_CN/LC_MESSAGES/global_settings.po]:
```suggestion
"**开发模式** 中，缓存编译过的运行时位码 (compiled runtime bitcode) 以节省启动时间："
```

File: locales/zh_CN/LC_MESSAGES/global_settings.po [hunk 10]
```diff
@@ -27,184 +28,179 @@ msgid "Backends"
...
 
 #: ../../global_settings.rst:48
 msgid "``ti.init(arch=ti.cuda)`` is equivalent to ``export TI_ARCH=cuda``."
-msgstr ""
+msgstr "``ti.init(arch=ti.cuda)`` 相当于 ``export TI_ARCH=cuda``."
 
 #: ../../global_settings.rst:49
 msgid "``ti.init(log_level=ti.TRACE)`` is equivalent to ``export TI_ARCH=trace``."
-msgstr ""
+msgstr "``ti.init(log_level=ti.TRACE)`` 相当于 ``export TI_ARCH=trace``."
```
Comment (archibate) [inline locales/zh_CN/LC_MESSAGES/global_settings.po]:
```suggestion
msgstr "``ti.init(log_level=ti.TRACE)`` 相当于 ``export TI_LOG_LEVEL=trace``."
```
Found a typo in original doc :rofl:

**Review by archibate — CHANGES_REQUESTED** [review]

**Review by isdanni — APPROVED** [review]
LGTM! thanks! 🚀

**Pull request merged**